pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -172,8 +206,10 @@ func (r *SubnetPortReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				MaxConcurrentReconciles: common.NumReconcile(),
 			}).
 		Watches(&vmv1alpha1.VirtualMachine{},
-				handler.EnqueueRequestsFromMapFunc(r.vmMapFunc),
-				builder.WithPredicates(predicate.LabelChangedPredicate{})).
+			handler.EnqueueRequestsFromMapFunc(r.vmMapFunc),
+			builder.WithPredicates(predicate.LabelChangedPredicate{})).
+		Watches(&v1alpha1.AddressBinding{},
+				handler.EnqueueRequestsFromMapFunc(r.addressBindingMapFunc)).
```

Comment:
indent?

Reply:
Run go fmt again, the indent is correct.